ext/msgpack/buffer.c

```diff
@@ -98,6 +98,20 @@ void msgpack_buffer_destroy(msgpack_buffer_t* b)
     }
 }
 
+size_t msgpack_buffer_memsize(const msgpack_buffer_t* b)
+{
+    size_t memsize = 0;
+    msgpack_buffer_chunk_t* c = b->head;
+
+    while(c) {
+        memsize += sizeof(msgpack_buffer_chunk_t);
+        memsize += sizeof(char) * (c->last - c->first);
```

Comment:
`sizeof(char)` is 1 by definition, as `sizeof` returns the size in terms of number of `chars`.

Reply:
Indeed, I added it for clarity, but if you think it just adds confusion I'm happy to remove it.